### DISCUSSION TITLE
2023-07-02-great

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#145] [General]
# 译·如何做出伟大的工作 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share最近「黑客与画家」的作者出了一篇新的文章《How To Do Great Work / 如何做出伟大的工作》，写得特别好，很符合我的胃口，我用 ChatGPT4 给翻译了一下，帮大伙省下 Token，全文很长，可以有空时候细看。

https://tw93.fun/2023-07-20/great.html

**Comment by ovws** [comment]
我怀疑有ai的嫌疑

**Reply by tw93** [reply]
哈哈 本文通过 ChatGPT4 翻译

**Comment by Laitr0n** [comment]
正文中的备注1、2、3、7、8、9、15-26标号缺失

**Reply by tw93** [reply]
还真是 那应该是被 ChatGPT 给吃了

**Comment by tsurumi-yizhou** [comment]
有英文原链接吗？锻炼下我的阅读能力

**Reply by moonscar** [reply]
http://www.paulgraham.com/greatwork.html
原文大约是这里

**Comment by onepisYa** [comment]
1 2 3 7 8 9 15 ~ 24 26 备注标号缺失

**Reply by tw93** [reply]
哈哈 你可以给我提pr改改改

**Comment by anguschiu1** [comment]
請教一下，怎樣可以用得到GPT4？GPT4 不是要用國外卡先付費成付費用戶才用得到嗎？
（若不方便透露方法，也請暗示一下研究的方向，謝謝！）

**Reply by tw93** [reply]
有大量的办法，包括找人帮你开一张美国的卡，不过很重要的需要保证你的网络代理是ok的，比如使用OpenAI所有服务一直稳定在美国
